packages/mobile/src/escrow/actions.ts

```diff
@@ -13,17 +20,21 @@ export interface EscrowedPayment {
   expirySeconds: BigNumber
 }
 
-// The number of seconds before the sender can revoke the payment.
-export const EXPIRY_SECONDS = 432000 // 5 days in seconds
+// The number of seconds before the sender can reclaim the payment.
+export const EXPIRY_SECONDS = 1
```

Comment:
I changed the default payment expiry time in the wallet from 5 days to 1 second so senders can instantaneously reclaim payments. Would appreciate thoughts/opinions!

Reply:
seems ok to me

Reply:
@nityas thoughts?

Reply:
Decision was to keep it at 5 days -- changed back.